snmp/datadog_checks/snmp/data/profiles/_generic-host-resources.yaml

```
@@ -65,3 +65,7 @@ metrics:
         column:
           OID: 1.3.6.1.2.1.25.3.3.1.1
           name: hrProcessorFrwID
+      - tag: deviceindex
+        column:
+          OID: 1.3.6.1.2.1.25.3.2.1.1
+          name: hrDeviceIndex
```

```suggestion
          # we can use `hrDeviceIndex` here since `hrDeviceEntry` (1.3.6.1.2.1.25.3.2.1) and `hrProcessorEntry` (1.3.6.1.2.1.25.3.3.1) share the same index `hrDeviceIndex`
          OID: 1.3.6.1.2.1.25.3.2.1.1
          name: hrDeviceIndex
```

I think we should add a comment to make it explicit why we can use a device index in processor table metrics